crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -2013,7 +2013,7 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshBindGroup<I> {
                 It should be set by the prepare_mesh_bind_group system.\n\
                 This is a bevy bug! Please open an issue."
             );
-            return RenderCommandResult::Failure;
+            return RenderCommandResult::Skip;
```

Comment:
should this one be a failure?

Reply:
Probably, I just did a batch rename of every `Failure` to `Skip` to keep the current behaviour

Reply:
done